shared/src/commonMain/kotlin/com/br/kmmdemo/forecasts/WeatherEnum.kt

```diff
@@ -0,0 +1,25 @@
+package com.br.kmmdemo.forecasts
+
+import com.br.kmmdemo.resources.SharedRes
+import dev.icerock.moko.resources.ImageResource
+
```

Comment:
Should these weather text be moved to the strings resources

Reply:
Good catch. @MMurrayAtBR Could you do a quick change to get the strings moved into resources?

Reply:
stringResource is a composable function that can not be called from a standard Enum. I can move them to constants, though